msbuild/Xamarin.MacDev.Tasks/Tasks/WriteItemsToFile.cs

```diff
@@ -89,5 +89,13 @@ public void Cancel ()
 			if (ShouldExecuteRemotely ())
 				BuildConnection.CancelAsync (BuildEngine4).Wait ();
 		}
+
+		//We are expecting the files to be already present in the Mac
+		public bool ShouldCopyToBuildServer (ITaskItem item) => false;
+
+		//We want empty output files to be created in Windows
+		public bool ShouldCreateOutputFile (ITaskItem item) => true;
```

Comment:
Remote tasks create empty output files by default for each `ITaskItem` output property, so this should not be needed.

Reply:
@emaf is there a deeper problem here? It's not working in this log:

[RecipeFinder_Debug_AnyCPU_net9.0-ios_Build_2025-05-05T16_03_13.6534537-05_00.zip](https://github.com/user-attachments/files/20064699/RecipeFinder_Debug_AnyCPU_net9.0-ios_Build_2025-05-05T16_03_13.6534537-05_00.zip)

Reply:
It seems to be copying the outputs and run the write items locally because of the missing SessionId, as Ema mentioned:

![image](https://github.com/user-attachments/assets/79b0a057-4339-40c1-aa3c-84f98f29e2ed)

Reply:
Going to leave this open and review what my test changes did.